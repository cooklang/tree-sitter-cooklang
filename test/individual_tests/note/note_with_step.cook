> Don't burn the roux!
Mash @potato{2%kg} until smooth.