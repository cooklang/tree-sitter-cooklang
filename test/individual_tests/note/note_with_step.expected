(recipe 
  (note)
  (step (ingredient (name) (amount (quantity) (units)))))